calibration/CTDMOG/CGINS-CTDMOG-13594__20150626.csv

```diff
@@ -20,6 +20,6 @@ serial,name,value,notes
 37-13594,CC_ptcb1,2.5e-05,
 37-13594,CC_ptcb2,0,
 37-13594,CC_ptempa0,151.513,
-37-13594,CC_ptempa1,-0.06444754,
+37-13594,CC_ptempa1,-6.444753e-02,
 37-13594,CC_ptempa2,-2.369059e-07,
-37-13594,CC_wbotc,1.5486e-07,
+37-13594,CC_wbotc,1.548578e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT